cli_tools/gce_windows_upgrade/upgrade_script_2008r2_to_2012r2.ps1

```diff
@@ -21,13 +21,13 @@ try {
   Remove-Item 'C:\$WINDOWS.~BT' -Recurse -ErrorAction Ignore
 
   $ver=[System.Environment]::OSVersion.Version
-  Write-Host "windows_upgrade_current_version=$($ver.Major).$($ver.Minor)"
   if ("$($ver.Major).$($ver.Minor)" -ne "6.1") {
     if ("$($ver.Major).$($ver.Minor)" -eq "6.3") {
       Write-Host "GCEMetadataScripts: The instance is already running Windows 2012R2!"
-      Return
+      Write-Host "GCEMetadataScripts: Rerunning upgrade.ps1 for post-upgrade step..."
```

Comment:
* Remove ellipsis
* Explain why the re-run needs to occur

Reply:
Removed. re-run is a post-upgrade step.